app/interactors/datapass_webhook/archive_current_token.rb

```diff
@@ -0,0 +1,15 @@
+class DatapassWebhook::ArchiveCurrentToken < ApplicationInteractor
+  def call
+    return if %w[archive].exclude?(context.event)
```

Comment:
Je trouve cette formulation plus claire:
```suggestion
    return unless %w[archive].include?(context.event)
```

Reply:
bon si c'est iso ça fait une bonne raison